content/news/articles/2025-06-09-metrics-opensearch.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: layouts/post
+tags: news
+title: New Metrics for OpenSearch/Elasticsearch Domain Monitoring and S3 Bucket Size
+date: 2025-06-10
+excerpt: "Track detailed performance metrics for your Cloud.gov domains and S3"
+---
+
+Cloud.gov has introduced comprehensive domain metrics to help teams monitor system performance and resource utilization. As part of this update, Cloud.gov has also implemented a new metric that allows teams to easily track the size of their S3 buckets.
+
+This update allows customers to access metrics and satisfy [M-21-31 guidelines][m-21-31] for log retention. Metrics are now available in the [Cloud.gov logging system][logs].
+
+## Available S3 Metrics
+
+### Storage and Capacity Metrics
+- **Bucket Size in Bytes**
```

Comment:
```suggestion
- **Bucket Size in Bytes**, metric name: `XXX`
```

Can we add the metric name to each bullet, which will make it easier for people to search them?